src/wizards/tokenSwapDealWizard/components/daoStageContent/daoStageContent.html

```diff
@@ -26,11 +56,25 @@ <h2 class="sidebarHeader heading heading3 title" data-test="section-title">${nam
         input-reference.bind="daoTreasuryAddressInput"
         label-info="This address will receive tokens from the token swap"
         label.bind="name +' Treasury Address'">
-        <form-address-input
+        <form-address-input if.bind="treasuryAddresses.length <= 1"
           disabled.bind="disabled"
           value.bind="data.treasury_address | checksumAddress & validate"
           view-model.ref="daoTreasuryAddressInput">
         </form-address-input>
+        <autocomplete-select else
```

Comment:
Component name is `autocomplete-select` and events in the View Model are under `DropdownEvent`.
Can you come up with one name for both?